src/test/scala/za/co/absa/commons/spark/SchemaUtilsSpec.scala

```diff
@@ -102,4 +103,25 @@ class SchemaUtilsSpec extends AnyFlatSpec with Matchers with BeforeAndAfterAll w
     SchemaUtils.diffSchema(schemaA, schemaB).isEmpty should be(true)
     SchemaUtils.diffSchema(schemaB, schemaA).isEmpty should be(true)
   }
+
+  behavior of "isSubset"
+
+  it should "be true for E in D but not vice versa" in {
```

Comment:
Unbelievably minor, but why `E` and `D`?

Reply:
At the top of the Spec there are JSONs A to D, from which I get the schema.